functions/src/logic/buildQueue/cleaner.ts

```diff
@@ -10,20 +8,22 @@ export class Cleaner {
   }
 
   public static async cleanUpBuildsThatDidntReportBack() {
-    const startedBuilds = await CiBuilds.getStartedBuilds();
+    // Note that the cronjob has a limited runtime, so we can only process so many requests to dockerhub.
```

Comment:
by `so many`, do we mean 6 in the current context? And if so, should we extract a constant to make this a bit more clear? Also, I see a default value of 30 defined in `getStartedBuilds`'s implementation. Should these values match, be the same, etc?